share/node-build/jxcore+sm-dev

```diff
@@ -0,0 +1,6 @@
+after_install_package() {
+  fix_jxcore_directory_structure
+}
+
+install_git jxcore-dev "https://github.com/jxcore/jxcore.git" "master" jxcore_spidermonkey standard
+install_package npm "https://s3.amazonaws.com/nodejx/npmjx310.tar.gz#1071b88dbacbe366ded078ad4ae4e387ec61fe8c1cf48e248f8b82924bb9d54f" jxcore_npm
```

Comment:
@ktrzeciaknubisa is there any chance of a download URL existing that just points to 'latest' npmjx?

This build definition (and the other one in this PR) are meant to be the jxcore-dev builds that just pull from the master branch on github. But the associated npmjx package has the version hardcoded, which means these build definitions will need to be updated each time a new npmjx package/version is released. Hopefully, `https://s3.amazonaws.com/nodejx/npmjx-latest.tar.gz` or just `https://s3.amazonaws.com/nodejx/npmjx.tar.gz` could be a thing?